rgd/geodata/urls.py

```diff
@@ -110,8 +110,13 @@
         name='tilemetadata',
     ),
     path(
-        'api/geodata/imagery/image_entry/<int:pk>/tiles/<int:z>/<int:x>/<int:y>.jpg',
+        'api/geodata/imagery/image_entry/<int:pk>/tiles/<int:z>/<int:x>/<int:y>.png',
```

Comment:
We're pulling in a JPEG every time, though, right?

Reply:
I changed the encoding to PNG in https://github.com/ResonantGeoData/ResonantGeoData/pull/284/commits/981a7d124412d3a4e30fa04771ae31fa8d8a2c2d with:

```py
tile_source = GDALFileTileSource(file_path, projection=projection, encoding='PNG')
```